<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingLED500Dialog</class>
 <widget class="QDialog" name="SettingLED500Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBoxComm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>74</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>COM1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM20</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>通信ポート</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>111</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QComboBox" name="comboBoxLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>74</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bar-0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-14</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spinBoxLBar2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxLBar4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxLBar5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxLBar3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxLBar0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxLBar6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxLBar1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>111</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QComboBox" name="comboBoxRight">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>74</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bar-15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bar-29</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRBar6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRBar0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRBar2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRBar4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRBar1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRBar3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRBar5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QCheckBox" name="checkBoxSyncMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>入力同期点灯</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>前方無効区間パルス</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxSyncForward">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxSyncPulsePerBar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LED送りパルス</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxSyncBackward">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>後方無効パルス</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSetAll">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set all</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxAll">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4095</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonOnOff">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ON/OFF</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTransfer">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetLeft">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetRight">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Right</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLeft">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4095</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRight">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4095</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Grad value</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbGradValue">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
